commitizen/commands/bump.py

```diff
@@ -157,7 +163,11 @@ def __call__(self):  # noqa: C901
         c = git.commit(message, args=self._get_commit_args())
         if c.return_code != 0:
             raise BumpCommitFailedError(f'git.commit error: "{c.err.strip()}"')
-        c = git.tag(new_tag_version)
+        c = git.tag(
+            new_tag_version,
+            annotated=self.bump_settings.get("annotated_tag", False)
```

Comment:
to enhance readbility, I would suggest make `self.bump_settings.get("annotated_tag", False) or or bool(self.config.settings.get("annotated_tag", False))` a variable. I read this line for a few times to understand